src/core/api/debug/buffer_graph.ts

```diff
@@ -96,8 +96,8 @@ export default class SegmentBufferGraph {
     let maximumPosition;
     if (maximumPoint - minimumPoint > BUFFER_WIDTH_IN_SECONDS) {
       if (currentTime === undefined) {
-        minimumPosition = minimumPoint;
         maximumPosition = maximumPoint;
+        minimumPosition = maximumPoint - BUFFER_WIDTH_IN_SECONDS;
```

Comment:
How is it possible to have `currentTime` to `undefined`?

If currentTime is `undefined` we may have an issue in
`minimumPoint = Math.min(currentTime, minimumPoint);` 
that may return `NaN` ?

Reply:
You're right it seems it cannot be `undefined` here. I will update

Reply:
done